x-pack/test/typings/hapi.d.ts

```diff
@@ -22,6 +21,5 @@ declare module 'hapi' {
     security?: SecurityPlugin;
     actions?: ActionsPlugin;
     alerting?: AlertingPlugin;
-    task_manager?: TaskManager;
```

Comment:
in `x-pack/typings/hapi.d.ts` this wasn't removed, but the type was changed to `LegacyTaskManagerApi` instead